CHANGELOG_WINBOX.md

```diff
@@ -219,4 +237,4 @@
 - make file copy & paste work between routers again;
 - make Enter key activate Connect if one of connect fields were active even when master password field has not been entered yet;
 - fixed Reconnect in RoMON mode with "Open In New Window" enabled;
-- other fixes.
\ No newline at end of file
+- other fixes.
```

Comment:
Please revert this change.

Reply:
I see that this line has changed but I don't see the change itself.
I took that file from the `master` branch, added information about 3.36 but same result.

```
-- other fixes.
+- other fixes.
```

Any advice?

Reply:
Seems like line ending problem. Anyway you can keep this change, because there were no empty line in the end of file.